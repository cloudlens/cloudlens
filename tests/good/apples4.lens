var totalApples = 0;

match{
    "(?<numApples>\d+) apples# numApples:int";
    "^(?<name>[^ ]*)"
}

stream{
    print(e.name);
}

stream when (e){
    totalApples += e.numApples || 0;
    if(totalApples > 10)
        print("We picked too many apples: " + totalApples);
}
