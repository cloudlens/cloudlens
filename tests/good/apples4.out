Alice
Nicolas
Olivier
We picked too many apples: 12
Mandana
We picked too many apples: 12
Marion
We picked too many apples: 12
Bernard
We picked too many apples: 14
